### ISSUE TITLE
Text changes to observations items in layer selector

### ISSUE THREAD

**Issue by ajrobbins** [WikiWatershed/model-my-watershed#1641]
Text TBD - to be provided by Stroud.

**Comment by ajrobbins** [comment]
Shares a point with https://github.com/WikiWatershed/model-my-watershed/issues/1644

**Comment by ajrobbins** [comment]
From Stroud:

Here are our changes to the text in the Observations tab:
​- Add a "Delaware River Basin only:​" header to the top of the list, similar to the "Streams" and "Boundary" headers that are under the Overlays tab
- Eliminate CRBCZO, SWRC and USCRN from the list. With one exception, they don't have live data, and we hope to consolidate and substantially update these in the next month, hopefully around the time we update the UI.
- Change DEOS to "Delaware Environmental Observing System"
- Change NOS/CO-OPS to "NOAA Tides and Currents", but on the popup, keep the "Data from" link as is (NOS/CO-OPS)
- Change USGS to "USGS National Water Information System"
- Change DRB Point Source to "EPA Permitted Point Sources", and add number of sites that we display, similar to other observations.
 
We would also like to swap the color coding for the points on the map to better indicate water vs. land stations:
​​DEOS should be the green that is presently used for NOS/CO-OPS
NOS/CO-OPS​ should be the blue that is presently used for DEOS 

So in summary, the Observations tab should look something like this:

Delaware River Basin only:
​​Delaware Environmental Observing System (18)
NOAA Tides and Currents (5)
USGS National Water Information System (214)
EPA Permitted Point Sources ([# of sites])

**Comment by ajrobbins** [comment]
This is a bit more extensive than I was anticipating. Unless you think we can do all of this in the 1 point allocated, we should spin off cards for next sprint (like for changing the color coding on the points).
